pyaedt/modeler/cad/Primitives.py

```diff
@@ -275,45 +587,4908 @@ def version(self):
+
+        vArg1 = ["NAME:Selections", "Selections:=", selections, "NewPartsModelFlag:=", "Model"]
+        vArg2 = ["NAME:SheetThickenParameters"]
+        vArg2.append("Thickness:="), vArg2.append(self._arg_with_dim(thickness))
+        vArg2.append("BothSides:="), vArg2.append(bBothSides)
+
+        self.oeditor.ThickenSheet(vArg1, vArg2)
+
+        if isinstance(objid, list):
+            obj_list = []
+            for objl in objid:
+                obj_list.append(self.update_object(objl))
+            return obj_list
+        return self.update_object(objid)
+
+    @pyaedt_function_handler()
+    def sweep_along_normal(self, obj_name, face_id, sweep_value=0.1):
+        """Sweep the selection along the vector.
+
+        Parameters
+        ----------
+        obj_name : list, str, int, :class:`pyaedt.modeler.Object3d.Object3d`
+            Name or ID of the object.
+        face_id : int or list
+            Face or list of faces to sweep.
+        sweep_value : float, optional
+            Sweep value. The default is ``0.1``.
+
+        Returns
+        -------
+        pyaedt.modeler.cad.object3d.Object3d
+
+        References
+        ----------
+
+        >>> oEditor.SweepFacesAlongNormal
+        """
+        if not isinstance(face_id, list):
+            face_id = [face_id]
+        selections = self.convert_to_selections(obj_name)
+        vArg1 = ["NAME:Selections", "Selections:=", selections, "NewPartsModelFlag:=", "Model"]
+        vArg2 = ["NAME:Parameters"]
+        vArg2.append(
+            [
+                "NAME:SweepFaceAlongNormalToParameters",
+                "FacesToDetach:=",
+                face_id,
+                "LengthOfSweep:=",
+                self._arg_with_dim(sweep_value),
+            ]
+        )
+
+        objs = self._all_object_names
+        self.oeditor.SweepFacesAlongNormal(vArg1, vArg2)
+        self.cleanup_objects()
+        objs2 = self._all_object_names
+        obj = [i for i in objs2 if i not in objs]
+        for el in obj:
+            self._create_object(el)
+        if obj:
+            if len(obj) > 1:
+                return [self.update_object(self[o]) for o in obj]
+            else:
+                return self.update_object(self[obj[0]])
+        return False
+
+    @pyaedt_function_handler()
+    def sweep_along_vector(self, objid, sweep_vector, draft_angle=0, draft_type="Round"):
+        """Sweep the selection along a vector.
+
+        Parameters
+        ----------
+        objid : list, str, int, :class:`pyaedt.modeler.Object3d.Object3d`
+            Name or ID of the object.
+        sweep_vector : float
+            List of ``[x1, y1, z1]`` coordinates or Application.Position object for
+            the vector.
+        draft_angle : float, optional
+            Draft angle in degrees. The default is ``0``.
+        draft_type : str
+            Type of the draft. Options are ``"Round"``, ``"Natural"``,
+            and ``"Extended"``. The default is ``"Round"``.
```

Comment:
```suggestion
            Type of the draft. Options are ``"Extended"``, ``"Natural"``,
            and ``"Round"``. The default is ``"Round"``.
```
Alphabetize options.